xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp

```diff
@@ -203,48 +197,47 @@ bool CGUIDialogPVRGuideInfo::OnMessage(CGUIMessage& message)
   return CGUIDialog::OnMessage(message);
 }
 
-void CGUIDialogPVRGuideInfo::SetProgInfo(const CFileItem *item)
+void CGUIDialogPVRGuideInfo::SetProgInfo(const EPG::CEpgInfoTagPtr &tag)
 {
-  *m_progItem = *item;
+  m_progItem = tag;
```

Comment:
Ok. Just pointed it out since CFileItem and the other 'tags' are by design not, and that makes it dangerous to just replace them with pointers.